templates/etcd-on-cluster-dashboard.jsonnet

```diff
@@ -1,73 +1,216 @@
   )
+);
+
+
+local cpu_usage = grafana.graphPanel.new(
+  title='CPU usage',
+  datasource='$datasource',
 ).addTarget(
   prometheus.target(
-    'sum(etcd_server_is_leader{pod=~"$pod",job=~"$cluster"}) by (instance)',
-    format='table',
-    instant=true,
+    'sum(avg_over_time(pod:container_cpu_usage:sum{container="",pod!="", namespace=~"openshift-etcd.*"}[5m])) BY (pod, namespace)',
+    legendFormat='-> {{ pod }}',
   )
+);
+
+local mem_usage = grafana.graphPanel.new(
+  title='Memory usage',
+  datasource='$datasource',
 ).addTarget(
   prometheus.target(
-    'sum(etcd_debugging_mvcc_db_compaction_keys_total{pod=~"$pod",job=~"$cluster"}) by (instance)',
-    format='table',
-    instant=true,
+    'sum(avg_over_time(container_memory_working_set_bytes{container="",pod!="", namespace=~"openshift-etcd.*"}[5m])) BY (pod, namespace)',
+    legendFormat='-> {{ pod }}',
   )
 );
 
-local etcd_members = grafana.singlestat.new(
-  title='Etcd Members',
+local traffic_in = grafana.graphPanel.new(
+  title='Traffic in bytes total',
   datasource='$datasource',
 ).addTarget(
   prometheus.target(
-    'sum(etcd_server_id{job="etcd"})',
-    legendFormat='members'
+    'sum(rate(container_network_receive_bytes_total{ container="POD", pod!= "", namespace=~"openshift-etcd.*"}[5m])) BY (namespace, pod)',
+    legendFormat='-> {{ pod }}',
   )
 );
 
-local etcd_has_leader = grafana.singlestat.new(
-  title='Etcd has a leader?',
+local traffic_out = grafana.graphPanel.new(
+  title='Traffic out bytes total',
+  datasource='$datasource',
+).addTarget(
+  prometheus.target(
+    'sum(rate(container_network_transmit_bytes_total{ container="POD", pod!= "", namespace=~"openshift-etcd.*"}[5m])) BY (namespace, pod)',
+    legendFormat='-> {{ pod }}',
+  )
+);
+
+local client_traffic_in = grafana.graphPanel.new(
+  title='Client traffic in',
+  datasource='$datasource',
+).addTarget(
+  prometheus.target(
+    'rate(etcd_network_client_grpc_received_bytes_total[5m])',
+    legendFormat='{{instance}} Client Traffic In',
+  )
+);
+
+local client_traffic_out = grafana.graphPanel.new(
+  title='Client traffic out',
   datasource='$datasource',
 ).addTarget(
   prometheus.target(
-    'max(etcd_server_has_leader{pod=~"$pod",job=~"$cluster"})',
+    'rate(etcd_network_client_grpc_sent_bytes_total[5m])',
+    legendFormat='{{instance}} Client Traffic Out',
+  )
+);
+
+local peer_traffic_in = grafana.graphPanel.new(
+  title='Peer traffic in',
+  datasource='$datasource',
+).addTarget(
+  prometheus.target(
+    'sum(rate(etcd_network_peer_received_bytes_total[5m])) by (instance)',
+    legendFormat='{{instance}} Peer Traffic In',
+  )
+);
+
+local peer_traffic_out = grafana.graphPanel.new(
+  title='Peer traffic out',
+  datasource='$datasource',
+).addTarget(
+  prometheus.target(
+    'sum(rate(etcd_network_peer_sent_bytes_total[5m])) by (instance)',
+    legendFormat='{{instance}} Peer Traffic Out',
+  )
+);
+
+local disk_opeations_latency = grafana.graphPanel.new(
```

Comment:
This is not used right? Also, I see that's also defined in disk_sync_duration

Reply:
You are correct. This one is sum, and the other is per-instance. I think per-instance is more useful so I will keep that one and remove this unused one.